lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
@soyuka yes but the problem is that you are mixing bindings of descriptor + CLOB and non descriptor + `SQLT_CHR` which does not happen in Doctrine because Doctrine doesn not use descriptors for `CLOB`